go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/go-openapi/runtime v0.0.0-20180131174916-09fac855d850 // indirect
 	github.com/go-openapi/spec v0.0.0-20180213232550-1de3e0542de6
 	github.com/go-openapi/strfmt v0.19.3
-	github.com/go-openapi/swag v0.19.5
+	github.com/go-openapi/swag v0.19.12
```

Comment:
commit 1 changed this from 0.19.12 to 0.19.5 and commit 2 changed it back again.